src/JWT.ts

```diff
@@ -322,7 +364,58 @@ export async function createJWT(
     }
   }
   const fullPayload = { ...timestamps, ...payload, iss: issuer }
-  return createJWS(fullPayload, signer, header, { canonicalize })
+  return createJWS(fullPayload, signer, header, { canonicalize }) as Promise<string>;
+}
+
+// TODO create TSDoc
+export async function createMultisignatureJWT(
+  payload: Partial<JWTPayload>,
+  { expiresIn, canonicalize }: Partial<JWTOptions>,
+  issuers: { issuer: string, signer: Signer, alg: string }[],
+): Promise<string> {
+  if (issuers.length === 0) throw new Error('invalid_argument: must provide one or more issuers')
+
+  const timestamps: Partial<JWTPayload> = {
+    iat: Math.floor(Date.now() / 1000),
+    exp: undefined,
+  }
+  if (expiresIn) {
+    if (typeof expiresIn === 'number') {
+      timestamps.exp = <number>(payload.nbf || timestamps.iat) + Math.floor(expiresIn)
+    } else {
+      throw new Error('invalid_argument: JWT expiresIn is not a number')
+    }
+  }
```

Comment:
shouldn't there be a default expire time ?

Reply:
it's not in the DID or VC standard, so no. I also wouldn't want to change anything in these external libraries like this, that aren't to do with the additions we want - multi-signatures.